From eeb948c6b7d33df481faabe95f25071942045feb Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@notvalid.com>
Date: Wed, 14 Oct 2026 16:37:25 -0700
Subject: [PATCH] Use current stream API in native AuxPoW block fixture

Retain exact serialized parent coinbase bytes while replacing the removed CDataStream constructor with DataStream.

diff --git a/src/test/util/setup_common.cpp b/src/test/util/setup_common.cpp
index 494a5cd29412a9f73ff7d9178981c38af5a4ad02..1dbe5fb9aefd55bff1245d736dd212523f463616 100644
--- a/src/test/util/setup_common.cpp
+++ b/src/test/util/setup_common.cpp
@@ -417,7 +417,7 @@ CBlock TestChain100Setup::CreateAuxPowBlock(
     std::vector<uint8_t> coinbaseBytes = ToByteVector(MERGE_MINE_PREFIX);
     coinbaseBytes.insert(coinbaseBytes.end(), hashChainRoot.begin(),
                          hashChainRoot.end());
-    CDataStream ss{0, 0};
+    DataStream ss{};
     ss << uint32_t(1 << chainMerkleBranch.size());
     ss << mergeMineNonce;
     Span<const uint8_t> span = MakeUCharSpan(ss);
